src/controllers/userControllers.ts

```diff
@@ -31,16 +31,35 @@ interface User {
   isSeller?: boolean;
 }
 
+const usersPerPage: number = process.env.USERS_PER_PAGE
```

Comment:
It is recommendable to have this as part of the request, usually query parameter, because that way, there is flexibility to change as opposed to the environment variable